### ISSUE TITLE
Using '~' as custom mapping

### ISSUE THREAD

**Issue by antoinerey** [ionutvmi/path-autocomplete#50]
Defining custom mappings with '@' works like a charm, but defining the exact same alias using '~' does not show any autocompletion.

```js
{
  // This works as expected, showing autocompletion
  "path-autocomplete.pathMappings": {
    "@": "${workspace}"
  },

  // This DOES NOT work, no autocompletion are shown
  "path-autocomplete.pathMappings": {
    "~": "${workspace}"
  }
}
```

I suspect this to conflict with the pre-defined home alias ('~'). 

**Should we actually be able to use '~' in custom mappings ?** 
**- If yes, what am I doing wrong ?**
**- If no, would you accept a PR to add this functionality ?**

**Comment by ionutvmi** [comment]
I just tried it locally and `"~": "${workspace}"` seems to work as expected.  
The mappings are processed before we reach the default aliases.

Can you please provide more details about your environment ? OS, vscode version, path autocomplete version, file type where you are trying to insert the path

**Comment by ionutvmi** [comment]
If you still find an issue on your local machine, yes a PR is always welcomed.

**Comment by antoinerey** [comment]
After a few more tries, I kinda got something.

When typing the string `~/src`, here's what's happening : 

**1.** I press `ALT` + `N` simultaneously (to get the `~` character)
**2.** The `~` character appears, but it's waiting for me to type another character. Depending on the key I'll press, it will combine into a single letter (`ñ` or `õ`), or it will be rendered as is (`~`)

That's where the magic appears !

**3a.** If I press `/`, I get the expected `~/`, but no autocomplete. The autocomplete will show after the next `/` (for exemple, `~/src/`).

**3b.** If I press `SPACE`, I still get the expected `~/`, but this time, the autocomplete shows up and everything works smoothly.

---

Basically, these exact keystrokes wont work

```
ALT + N       # ~
SHIFT + /     # ~/
```

But those works as expected 

```
ALT + N       # ~
SPACE         # ~
SHIFT + /     # ~/
```

--- 

I'm using : 

* VSCode 1.25.1
* macOS 10.13.4
* An AZERTY keyboard

**Comment by ionutvmi** [comment]
Thank you for investigating this.
I will consider this issue resolved.
